### DISCUSSION TITLE
Why Euler angles detected in Mtex differ from Channel 5

### DISCUSSION THREAD

**Discussion by cheneyjin** [mtex-toolbox/mtex#1321] [Ask Anything] [answered]
Hello All,

When I opened EBSD demo file 'Forsterite.ctf' both in Mtex and HKL channel 5 Tango, I found the Euler angles of the same grain are different. Is there any mistakes in settings? 
I took the X-axis to east and Z intoPlane and I got: 

<img width="783" alt="comp" src="https://user-images.githubusercontent.com/61216809/150701296-cb396ed8-80e8-45b3-8d1a-54ddd2916efe.png">

The Euler angles in Mtex are  (60.8, 91.3, 212.5)
but in Channel 5 are (119.2, 88.9, 32.7)

Cheney

**Comment by lfgmorales** [comment, marked as answer]
Hi

This is due to a rotation of 180 degrees around the normal of the sample surface, this has been discussed here previously, but you may want to have a look here

http://min.kueps.kyoto-u.ac.jp/SEM-EBSD/JpGU2016_SMP-P05_e.pdf

Or here

https://meetingorganizer.copernicus.org/EGU2016/EGU2016-8221.pdf

Cheers

Luiz

Luiz F. G. Morales
ScopeM - ETH Zürich
Sent from iPhone

On 24 Jan 2022, at 00:00, cheneyjin ***@***.***> wrote:

﻿

Hello All,

When I opened EBSD demo file 'Forsterite.ctf' both in Mtex and HKL channel 5 Tango, I found the Euler angles of the same grain are different. Is there any mistakes in settings?
I took the X-axis to east and Z intoPlane and I got:

[comp]<https://user-images.githubusercontent.com/61216809/150701296-cb396ed8-80e8-45b3-8d1a-54ddd2916efe.png>

The Euler angles in Mtex are (60.8, 91.3, 212.5)
but in Channel 5 are (119.2, 88.9, 32.7)

—
Reply to this email directly, view it on GitHub<https://github.com/mtex-toolbox/mtex/discussions/1321>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ALE5IN72RKT56EYYHVDGCFTUXSCAZANCNFSM5MT5V6MQ>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

[ { ***@***.***": "http://schema.org", ***@***.***": "EmailMessage", "potentialAction": { ***@***.***": "ViewAction", "target": "https://github.com/mtex-toolbox/mtex/discussions/1321", "url": "https://github.com/mtex-toolbox/mtex/discussions/1321", "name": "View Discussion" }, "description": "View this Discussion on GitHub", "publisher": { ***@***.***": "Organization", "name": "GitHub", "url": "https://github.com" } } ]

**Reply by cheneyjin** [reply]
> Hi This is due to a rotation of 180 degrees around the normal of the sample surface, this has been discussed here previously, but you may want to have a look here http://min.kueps.kyoto-u.ac.jp/SEM-EBSD/JpGU2016_SMP-P05_e.pdf Or here https://meetingorganizer.copernicus.org/EGU2016/EGU2016-8221.pdf Cheers Luiz 

Dear Luiz,
Thank you very much for your reply.
According to your reference, there is definitely a 180 degree difference between Channel5 and Mtex. But I'm still confused which one is correct ? Or since both the .ctf file are coming from Oxford instruments, so, neither of them is correct. What's your suggestion to get the true orientations?  Rotation 180 degree around Z in Channel5 or Mtex?

Best regards
Cheney

**Comment by cheneyjin** [comment]
It is corrected by:
`rot=rotation('Euler', 0*degree, 180*degree, 0*degree);`
`ebsd=rotate(ebsd,rot,'keepXY');`

**Reply by labnewman** [reply]
i did in the same situation with @cheneyjin. 
Still the same after use
rot=rotation('Euler', 0*degree, 180*degree, 0*degree);
ebsd=rotate(ebsd,rot,'keepXY');

is there any way to let the euler angle same with Tango?
